recipes/pyspark/pysparkretailapp/__init__.py

```diff
@@ -15,7 +15,7 @@
 # from Adobe.
 #####################################################################
 import sys
-sys.path.append("/usr/bin/anaconda/envs/dswpy36/lib/python3.6/site-packages/")
+sys.path.append("/usr/bin/anaconda/envs/dswpy36/lib/python3.7/site-packages/")
```

Comment:
Is this even used anymore?  I think this was for HDI back in the day?  I think it's fine but not sure this directory even exists anymore.